specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json

```diff
@@ -194,6 +194,54 @@
           }
         }
       },
+      "delete": {
+        "tags": [
+          "HanaOnAzure"
+        ],
+        "operationId": "HanaInstances_Delete",
+        "x-ms-long-running-operation": true,
+        "summary": "Deletes a SAP HANA instance.",
+        "description": "Deletes a SAP HANA instance with the specified subscription, resource group, and instance name.",
+        "x-ms-examples": {
+          "Get properties of a HANA instance": {
+            "$ref": "./examples/HanaInstances_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/HanaInstanceNameParameter"
+          }
+        ],
+        "responses": {
```

Comment:
204 response needs to be added/supported. If the request was well formed by there was nothing to be deleted should result in 204 instead of 200.

Reply:
@dwj300 I'm going to send a PR to the RP code to support this.

Reply:
Done

Reply:
@ravbhatnagar Please review